### PR TITLE
Am123456789 patch 1

### PR DESCRIPTION
**Pull request by AM123456789**

### CODE REVIEW

File: CensusPopulationByState.r [hunk 1]
```diff
@@ -0,0 +1,21 @@
+install.packages("tidycensus")
```
Comment (opencedar) [inline CensusPopulationByState.r]: Don't put install.packages in your code. You want to do this only once, in your terminal. You only need to install.packages once (from CRAN). Please delete these three lines

File: CensusPopulationByState.r [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+install.packages("tidyverse")
+install.packages("ggplot2")
+
+
+library(tidycensus)
+library(tidyverse)
+library(ggplot2)
+
+census_api_key("6f46a7532e0317f2318ad51b2aa17a2643edcc72", install = TRUE, overwrite=TRUE)
```
Comment (opencedar) [inline CensusPopulationByState.r]: Don't put your api key in your code. Please see comment in previous review. Research how to store keys as SYSTEM variables, and then call those in your code. You don't want other users using your API keys, even if they're "free" like the census one is.

File: CensusPopulationByState.r [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+
+#v17 <- load_variables(2021, "acs1", cache = TRUE)
+
+
+#2017-2021 State Populations
+Statei <- get_acs(geography ="state",variables = "B01001A_012")
```
Comment (opencedar) [inline CensusPopulationByState.r]: I don't know what this variable is. Please add a comment stating what it is in English.

File: CensusPopulationByState.r [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+View(Statei)
+#Plot
+ggplot (Statei, aes(x = estimate, y = NAME)) + geom_point()
```
Comment (opencedar) [inline CensusPopulationByState.r]: Your ggplot's x-axis says "estimate". It should list the variable name. The y-axis is "NAME" but they look like states to me. Also, was hoping to see a map with colors representing the proportion we are interested in. Please check out the package choroplethr.

**Review by opencedar — COMMENTED** [review]
Still need a readme.md file.
See inline code comments.

**Comment by alamstein** [comment]
Since we are moving towards just having a single .qmd file in the repo, I think it is best to close this PR for clarity.

**Pull request closed without merging**